packages/location-field/src/stories/Mobile.story.tsx

```diff
@@ -113,33 +112,34 @@ const StyledLocationField = styled(LocationField)`
 
 export default {
   title: "LocationField/Mobile Context",
-  component: LocationField
+  component: LocationField,
+  decorators: [intlDecorator]
 };
 
-export const Blank = () => (
+export const Blank = (): JSX.Element => (
   <LocationField
     currentPosition={currentPosition}
     geocoderConfig={geocoderConfig}
     getCurrentPosition={getCurrentPosition}
     locationType="from"
     onLocationSelected={onLocationSelected}
-    static
+    isStatic
```

Comment:
Sort props after rename (multiple instances)

Reply:
Addressed in 18d5606109e7dfce7d00d42b42c8b1b6f075a0bc